homeassistant-supervised/etc/systemd/resolved.conf

```diff
@@ -1,25 +0,0 @@
-#  This file is part of systemd.
-#
-#  systemd is free software; you can redistribute it and/or modify it
-#  under the terms of the GNU Lesser General Public License as published by
-#  the Free Software Foundation; either version 2.1 of the License, or
-#  (at your option) any later version.
-#
-# Entries in this file show the compile time defaults.
-# You can change settings by editing this file.
-# Defaults can be restored by simply deleting this file.
-#
-# See resolved.conf(5) for details
-
-[Resolve]
-#DNS=
-#FallbackDNS=1.1.1.1 8.8.8.8 1.0.0.1 8.8.4.4 2606:4700:4700::1111 2001:4860:4860::8888 2606:4700:4700::1001 2001:4860:4860::8844
-#Domains=
-DNSSEC=no
-DNSOverTLS=no
-#MulticastDNS=yes
-#LLMNR=yes
-#Cache=yes
-DNSStubListener=no
```

Comment:
This setting most likely came from HAOS too, particularly this commit: https://github.com/home-assistant/operating-system/commit/7c25f7c18723841d7b558543612cab18c93d3910

To be safe, we should keep this config by default, but as @Yayg mentions, in a file at etc/systemd/resolved.conf.d/.